abis_mapping/templates/survey_occurrence_data/schema.json

```diff
@@ -319,7 +330,18 @@
       "title": "Other Catalog Numbers",
       "description": "...",
       "example": "BHP2012-7521 | M12378",
-      "type": "list",
+      "type": "string",
```

Comment:
type string but example shows a list type
also in incidental occurrences (V2) otherCatalogNumbers is a list.

Reply:
There are some mismatches between the schema of incidental and SSD shared fields.
Still merging that but created a ticket for harmonisation of the two schema